conformance/lang/lang-library/lang.regexp/find.balt

```diff
@@ -0,0 +1,92 @@
+Test-Case: output
+Description: Test `find` function.
+
+function init() {
+    var regExpr1 = re `Here`;
```

Comment:
Most of the regexp literals are just strings. Please use more complex variations whenever possible so that you can check the functionality as well. 
Keep in mind how ballerina spec differed when writing the regex combinations https://ballerina.io/spec/lang/master/#section_10.1

> The semantics of a Ballerina regular expression are as specified by ECMAScript subject to the following points:
the version of ECMAScript used is ECMAScript 2022 (ECMA-262, 13th edition);
Ballerina uses the semantics defined by ECMAScript for when Unicode mode, as set by the u flag, is in effect (ECMAScript regular expressions work differently depending on whether Unicode mode is in effect or not);
the semantics of \p and \P, and of the i flag depend on the version of Unicode being used; this version of Ballerina uses Unicode version 10;
regular expression constructs that depend on the definition of a line terminator or whitespace use the Ballerina definitions of those rather than the ECMAScript definitions; specifically
. matches any character other than 0xA or 0xD;
\s matches any character allowed by the WhiteSpaceChar production; as in ECMAScript, \S matches any character not matched by \s;
the semantics of ReFlagsOnOff (which is not supported in ECMAScript) are as follows:
the flags are modified within the parenthesized group containing the ReFlagsOnOff, with those in ReFlagsOn being turned on, and those in ReFlagsOff being turned off;
it is an error for any flag to be specified more than once within ReFlagsOnOff;
the meaning of the m, s and i flags are as in ECMAScript;
the meaning of the x flag is as in Perl: when this flag is in effect, any character in the regular expression that matches to the WhiteSpaceChar production, but is not within an ReCharSet is ignored; in addition, comments starting with # and continuing to the end of the line are ignored.

Reply:
Better to add some reasonable regex pattern like find any email, webpage and etc. WDYT?

Reply:
Will test with both complex / commonly used patterns.